src/web/styles.scss

```diff
@@ -1,5 +1,9 @@
 /* You can add global styles to this file, and also import other style files */
 
+html {
+  scroll-behavior: auto !important;
```

Comment:
yeah of course.

Reply:
@samuelfangjw as i mentioned earlier in the discussion, we need to use scroll behavior auto so that ngx-scroll-page would work. I will add an explanation on the code and push it.

Reply:
@samuelfangjw it is done.